packages/ckeditor5-dev-env/tests/release-tools/utils/transform-commit/transform-commit-utils.js

```diff
@@ -92,6 +92,11 @@ describe( 'dev-env/release-tools/utils/transform-commit', () => {
 				expect( transformCommit.getCommitType( 'Fix' ) ).to.equal( 'Bug fixes' );
 				expect( transformCommit.getCommitType( 'Other' ) ).to.equal( 'Other changes' );
 			} );
+
+			it( 'should support aliases for the "Fix" type', () => {
+				expect( transformCommit.getCommitType( 'Fixes' ), 'Fixes' ).to.equal( 'Bug fixes' );
```

Comment:
You have duplicated `Fixes` after the comma.
Same below.

Reply:
Yes because I would like to see which assertion fail. If the first one will fail, in the console I will see "Fixes" which will indicate a proper scenario.

Reply:
Wow. I didn't know that. 👍